protoc_plugin/mono_pkg.yaml

```diff
@@ -12,4 +12,6 @@ stages:
       - command: ./../tool/setup.sh
       - command: make protos
       - test
+      - test: -p chrome
```

Comment:
I don't know that we need to run the protoc_plugin tests in the browser (but haven't examined the tests).

Reply:
Tests that need proto files are in `protoc_plugin/test`, so lots of useful tests there. We have all kinds of tests, encoding, decoding, API, ... 7,434 lines of tests in protoc_plugin vs. 1,474 lines (excluding comments, whitespace, and generated files).